NDB.Covid19/NDB.Covid19.iOS/Views/Welcome/ChildViews/PageViewController.cs

```diff
@@ -50,5 +51,27 @@ protected void InitSubTitle(UILabel label, string text)
             InitLabel(label, FontType.FontBold, text, 16, 22);
             label.AccessibilityAttributedLabel = AccessibilityUtils.RemovePoorlySpokenSymbols(text);
         }
+
+        /// <summary>
```

Comment:
Interesting, did you just try to set it as subclass to BaseViewController?

Reply:
No, didn't think of that, but that is better. Seems to behave the same way. Fixed now.